reactor-netty-core/src/main/java/reactor/netty/udp/UdpClient.java

```diff
@@ -135,6 +135,10 @@ public final UdpClient metrics(boolean enable) {
 		return super.metrics(enable);
 	}
 
+	public final UdpClient metrics(boolean enable, String prefix) {
+		return metrics(enable, () -> new UdpClientConfig.MicrometerUdpClientMetricsRecorder(prefix));
+	}
+
```

Comment:
Is this needed? One can use 
`reactor.netty.transport.Transport#metrics(boolean, java.util.function.Supplier<? extends reactor.netty.channel.ChannelMetricsRecorder>)` with `MicrometerChannelMetricsRecorder`